frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -103,21 +105,19 @@ function PlayerResultsItem(props: PlayerResultsCardProps) {
       return '0';
     }
 
-    const percent = Math.round((score / 12321) * 100);
+    const percent = Math.round((score / numProblems) * 100);
     return `${percent}%`;
   };
 
   const getSubmissionTime = () => {
-    if (!bestSubmission) {
-      return 'N/A';
+    if (!time) {
+      return 'Never';
     }
 
-    // Calculate time from start of game till best submission
-    const startTime = new Date(gameStartTime).getTime();
-    const diffMilliseconds = new Date(bestSubmission.startTime).getTime() - startTime;
+    const currentTime = new Date().getTime();
+    const diffMilliseconds = currentTime - new Date(time).getTime();
     const diffMinutes = Math.floor(diffMilliseconds / (60 * 1000));
-
-    return ` ${diffMinutes} min`;
+    return `${diffMinutes}m ago`;
```

Comment:
I see this was changed from how long from the game it took to submit, to how long ago they submitted. I'm open to discussion on this - I feel like whereas the "ago" might be more intuitive in the short-term, it's not quite as useful and how long it took to find the correct submission may actually be a better metric. Thoughts @jasonz6688 and @alankbi?

Reply:
Yeah I think this should prob remain as how long it took to submit it - main reason being that on the results page (and in reports) it kinda loses meaning if the time keeps increasing after the game is already over.